<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PrivacyRule</class>
 <widget class="QWidget" name="PrivacyRule" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Privacy Rule</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Then:</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="2" column="2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="ck_messages" >
         <property name="text" >
          <string>Messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ck_queries" >
         <property name="text" >
          <string>Queries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ck_presenceOut" >
         <property name="text" >
          <string>Outgoing Presence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ck_presenceIn" >
         <property name="text" >
          <string>Incoming Presence</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="cb_action" />
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="cb_type" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>If:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3" >
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>61</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>51</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QComboBox" name="cb_value" >
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3" >
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
